plugins/imagebase/plugin.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+( function() {
+	'use strict';
+
+	function createWidgetDefinition( editor, definition ) {
+		var defaultTemplate = new CKEDITOR.template(
+			'<figure class="{basicClass}">' +
```

Comment:
Is `basicClass` defined anywhere or is it just a leftover?

Reply:
It's a leftover from the previous version, it could be safely removed.